projects/packages/search/src/class-rest-controller.php

```diff
@@ -216,6 +236,55 @@ public function get_search_results( $request ) {
 		return $this->make_proper_response( $response );
 	}
 
+	/**
+	 * Activate plan: activate the search module, instant search and do initial configuration.
+	 * Typically called from WPCOM.
+	 *
+	 * POST `jetpack/v4/search/plan/activate`
+	 *
+	 * @param WP_REST_Request $request - REST request.
+	 */
+	public function activate_plan( $request ) {
+		// Update plan data, plan info is in the request body.
+		// We do this to avoid another call to WPCOM and reduce latency.
+		$plan_info = $request->get_json_params();
+		if ( ! $this->plan->set_plan_options( $plan_info ) ) {
+			return new WP_Error( 'invalid_request', esc_html__( 'Plan info does not exist in request', 'jetpack' ), array( 'status' => 400 ) );
```

Comment:
```suggestion
			return new WP_Error( 'invalid_request', esc_html__( 'Plan info could not be fetched from WordPress.com', 'jetpack' ), array( 'status' => 400 ) );
```

Reply:
Addressed in af033089bf. Changed to `Plan info could not be fetched from Jetpack.com` - WordPress.com might not be clear.